src/main/groovy/val/Check.groovy

```diff
@@ -125,10 +125,10 @@ abstract class AbstractComposedCheck implements Check {
 class AndCheck extends AbstractComposedCheck {
     @Override
     ResultMap call(Object input, EvalContext ctx) {
-        ResultMap result = ResultMap.passed()
+        ResultMap result = ResultMap.CLEAN
```

Comment:
passed was renamed to clean as the framework could also be used to return warnings in which case it may still "pass" with values.